### PR TITLE
Update the readme to be more detailed.

### PR DESCRIPTION
**Pull request by nathanbullock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
-# CCCprevious
-jkhgkgh
+# Canadian Computing Competition
+
+This repository is for solutions to past Canadian Computing Competition questions.
+
+https://cemc.math.uwaterloo.ca/contests/computing.html
```
Comment (lillianbullock) [inline README.md:5]: hello test

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
-# CCCprevious
-jkhgkgh
+# Canadian Computing Competition
```
Comment (lillianbullock) [inline README.md:1]: foood is goo

File: README.md (hunk 1 above)
Comment (nathanbullock) [inline README.md:5]: Done

File: README.md (hunk 2 above)
Comment (nathanbullock) [inline README.md:1]: Done

**Pull request merged**